assets/src/story-embed-block/block/transforms.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const transforms = {
+  from: [
+    {
+      type: 'shortcode',
+      tag: 'web_stories_embed',
+      attributes: {
+        align: {
+          type: 'string',
+          default: 'none',
+          shortcode: ({ named: { align } }) => {
+            return align;
+          },
+        },
+        height: {
+          type: 'number',
+          default: 600,
```

Comment:
Are default values supported here? Don't see that in the docs

Reply:
You know, I saw defaults in example and put them in to test if they work and forget to take them out. Give a sec I will take them out.

Reply:
Ah, alright! Thanks!